src/internal/ArbitraryBase.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * An instance of `Schemable` for `fast-check` arbitraries that emit valid values
  *
- * @since 0.0.2
+ * @since 1.0.0
```

Comment:
any reason you change all the `since`s?

Reply:
Once 1.0.0 releases, since it's the first proper release, I think all the `since`'s should be updated to 1.0.0.  I've just been doing it bit by bit